### PR TITLE
[GEOS-7576] Make rescale visitors rescale polygons graphic-margin parameter

### PR DESCRIPTION
**Pull request by nmco**
Associated issue:
https://osgeo-org.atlassian.net/browse/GEOS-7576

### CODE REVIEW

File: modules/library/api/src/main/java/org/geotools/styling/PolygonSymbolizer.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *    GeoTools - The Open Source Java GIS Toolkit
  *    http://geotools.org
  * 
- *    (C) 2002-2008, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2002-2016, Open Source Geospatial Foundation (OSGeo)
```
Comment (aaime) [inline modules/library/api/src/main/java/org/geotools/styling/PolygonSymbolizer.java]: No need to update the copyright dates anymore, just add a standard header with the current year if the file is new or does not have a header

File: modules/library/main/src/main/java/org/geotools/styling/visitor/UomRescaleStyleVisitor.java [hunk 2]
```diff
@@ -197,6 +197,7 @@ public void visit(PolygonSymbolizer poly) {
         Unit<Length> uom = copy.getUnitOfMeasure();
         rescaleStroke(copy.getStroke(), uom);
         rescaleFill(copy.getFill(), uom);
+        scaleIntArrayOption(copy.getOptions(), TextSymbolizer.GRAPHIC_MARGIN_KEY, uom);
```
Comment (aaime) [inline modules/library/main/src/main/java/org/geotools/styling/visitor/UomRescaleStyleVisitor.java]: While it's the same value, it's confusing to see TextSymbolizer referenced here, PolygonSymbolizer gets the right constant in this same pull request.

File: modules/library/api/src/main/java/org/geotools/styling/PolygonSymbolizer.java (hunk 1 above)
Comment (aaime) [inline modules/library/api/src/main/java/org/geotools/styling/PolygonSymbolizer.java]: Mind, this means you don't need to change the commit either, leave them be, just remember that you don't need to alter them anymore for the future changes.

File: modules/library/main/src/main/java/org/geotools/styling/visitor/UomRescaleStyleVisitor.java (hunk 2 above)
Comment (nmco) [inline modules/library/main/src/main/java/org/geotools/styling/visitor/UomRescaleStyleVisitor.java]: Ouch ... I added the PolygonSymbolizer constant to make it clear and forgot to updated it (facepalm).

**Comment by jodygarnett** [comment]
Judging from the above this is not yet ready to merge...

**Comment by nmco** [comment]
Updated.

**Pull request merged**